### PR TITLE
Rewords various objectives and makes 'Follow the inviolate' always succeed

### PR DESCRIPTION
**Pull request by LittleBigKid2000**
I've reworded the description of various objectives to make them more clear.

- Hijack's description is more clear about people escaping on pods not counting and that antags (Other than ERTs), cyborgs, and simple animals don't count.
- The AI equivalent of hijack ("Do not allow any organic lifeforms to escape on the shuttle alive.") counts IPCs as organic lifeforms. The description has been changed accordingly.
- Vox's steal/trade objective is more clear about tasers and disablers counting as guns.
- Vox's inviolate objective will always succeed, unless an admin decides otherwise.

### CODE REVIEW

**Comment by LittleBigKid2000** [comment]
Shit, it doesn't compile. DNM for now.

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -311,7 +311,7 @@ var/list/potential_theft_objectives=subtypesof(/datum/theft_objective) \
 		return 0
 
 /datum/objective/block
-	explanation_text = "Do not allow any organic lifeforms to escape on the shuttle alive."
+	explanation_text = "Do not allow any organic lifeforms or IPCs to escape on the shuttle alive."
```
Comment (FlattestGuitar) [inline code/game/gamemodes/objective.dm]: "Do not allow any lifeforms, be it organic or synthetic to escape on the shuttle alive." would sound better. Just saying.

Comment (LittleBigKid2000) [inline code/game/gamemodes/objective.dm]: But synthetic lifeforms would include cyborgs

Comment (FlattestGuitar) [inline code/game/gamemodes/objective.dm]: Organic lifeforms would include diona nymph and bacteria.

**Comment by monster860** [comment]
While you're at it, you should probably make it so laser tag guns don't count as objectives.

**Comment by LittleBigKid2000** [comment]
I have no idea how I could (easily) go about making laser tag guns not count, and Vox not knowing the difference between laser tag guns and actual laser guns would be funny.

**Comment by Davdichan** [comment]
@monster860 HELL NO. Vox can use laser tag guns! Like shit who says they are gonna test these things.

**Comment by Lady-Luck** [comment]
Can we just remove Vox Raiders from rotation instead

**Comment by Davdichan** [comment]
@Lady-Luck Only if they become side-antags/random event for midround antags.

**Pull request merged**